src/plugins/data/common/index_patterns/index_patterns/index_pattern.ts

```diff
@@ -251,7 +255,6 @@ export class IndexPattern implements IIndexPattern {
    * @param name field name
    * @param script script code
    * @param fieldType
-   * @param lang
```

Comment:
my IDE was complaining that this is not a param, so might as well remove it even though its not in scope of my work